frontend/src/components/profile/profile.tsx

```diff
@@ -1,47 +1,74 @@
 import * as React from 'react';
-import { SideMenu, UserInfo, Documents } from './components';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router';
+import { SideMenu, UserInfo, Documents, Diagnoses } from './components';
 import { EditUserPopup } from 'components/common';
-import { IEditUserPayload, IUserTypeDoctor } from 'common/interfaces';
+import {
+  IEditUserPayload,
+  IUserTypeDoctor,
+  IUserWithPermissions,
+} from 'common/interfaces';
 import { UserType } from 'common/enums';
-import { useSelector } from 'react-redux';
-import { useDispatch } from 'react-redux';
 import { RootState } from 'common/types';
-import { useParams } from 'react-router';
 import { ProfileActionCreator } from 'store/slices';
 
 import styles from './styles.module.scss';
+import { ProfileTab } from 'common/enums/profile';
 
 type RouteParam = {
   id: string;
 };
 
 const Profile: React.FC = () => {
   const [isEditMode, setIsEditMode] = React.useState<boolean>(false);
+  const [profileTab, setProfileTab] = React.useState<ProfileTab>(
+    ProfileTab.PERSONAL_INFO,
+  );
+
   const { user } = useSelector(({ profile }: RootState) => ({
     user: profile.user,
   }));
   const dispatch = useDispatch();
   const { id } = useParams<RouteParam>();
   const isDoctor = user?.type === UserType.DOCTOR;
+
+  const getProfileTab = (tab: ProfileTab) => {
+    switch (tab) {
+      case ProfileTab.DIAGNOSES: {
+        return <Diagnoses user={user as IUserWithPermissions} />;
+      }
+
+      default:
+        return (
+          <UserInfo
+            user={user as IUserWithPermissions}
```

Comment:
```suggestion
        return <Diagnoses user={user} />;
      }

      default:
        return (
          <UserInfo
            user={user}
```
User with permissions should be as default in the reducer